keyboards/40percentclub/25/config.h

```diff
@@ -19,7 +19,7 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x3430
```

Comment:
This vendor ID is already in use and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x3430&p=&n=

Ideally, anything above 0x4000 is fairly safe, but the higher, the better.

Reply:
Picked 0x4025, which can read as 40 %